addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -153,6 +153,12 @@ class ComposerTextInput extends Component {
      * @private
      */
     _update() {
+        if (document.activeElement === this._textareaRef.el && !this.composer.hasFocus) {
+            // Needed to ensure the right composer is focused when the composer changes.
```

Comment:
This needs clarification. I think it can more simply be expressed by "Ensure focus state in composer record is kept in sync with actual focus in DOM. Indeed OWL might re-use the same component for displaying a different record."

But now that I write it, I think this is actually a mistake. I don't think the focus should be automatically kept like that on switching thread/composer. There is a "re-focus" because we do call "focus" when opening a new thread, but this shouldn't be done automatically at component level (what if we didn't want the focus in models??).

Can we find a way (t-key?) to instruct OWL to destroy it/remove its focus by default? Or do we have to live with this?

Reply:
I don't know, I don't know OWL deep enough to answer your last question...
I'll just change the comment for now.

Reply:
Just try with a `t-key` based on `composer.localId` I guess and if that doesn't work then it's probably not possible?

Reply:
Seems to work